src/main.ts

```diff
@@ -298,6 +304,19 @@ module.exports = function (argv: string[]): void {
 		.option('-f, --force', 'Skip confirmation prompt when unpublishing an extension')
 		.action((id, { pat, azureCredential, force }) => main(unpublish({ id, pat, azureCredential, force })));
 
+	program
+		.command('generate-manifest')
+		.description('Generates the extension manifest from the provided VSIX package.')
+		.requiredOption('-i, --packagePath <path>', 'Path to the VSIX package')
+		.option('-o, --out <path>', 'Output the extension manifest to <path> location (defaults to .signature.manifest)')
```

Comment:
As far as I can understand, the fact that `.signature.manifest` is the default value is an implementation detail of vsce-sign. If we declare it here, I suggest to also enforce it here. I also think it's weird that `.signature.manifest` is the default name. Wouldn't `extension.manifest` be a better name, since it has nothing to do with signatures at this point?

```suggestion
		.option('-o, --out <path>', 'Output the extension manifest to <path> location (defaults to .signature.manifest)', 'extension.manifest')
```

Reply:
I will call it like VSIX. `(defaults to <name>-<version>.vsix)`